### DISCUSSION TITLE
Deleting parts of an audio track during audio playback

### DISCUSSION THREAD

**Discussion by andrio1988** [audacity/audacity#4874] [Ideas & Feature Requests]
Guys, please add the ability to delete selections of edited audio during playback. For example, while listening to an audio recording, I delete silence sections and other parts of the audio in advance without stopping playback. This feature is implemented in Wavelab Cast and Pro. It is very missing to speed up the workflow when working with long audio files.

**Comment by petersampsonaudacity** [comment]
As a workaround while playing _(or even recording too)_ I would 

1. click and drag to select the region to be deleted later
2. use **Ctrl+B** to create a range label for the later deletion
3. repeat if necessary for other parts to be deleted later
4. once playback has stopped, click on the label's horizontal bar to re-select the audio 
5. Press **Delete** or **Ctrl+K** to delete the selection

One of the advantages of this technique is that you can carefully adjust the boundaries of the labelled region(s) before making the deletion for more accurate editing.

_Just be extra careful if you use this technique during recording !    
I used to label the inter-song gaps while I was recording my LPs and tapes - you just have to make sure you don't inadvertently interrupt the recording with a Stop shortcut._
